kairos-cli/src/crypto/public_key.rs

```diff
@@ -0,0 +1,24 @@
+use crate::crypto::error::CryptoError;
+use casper_types::bytesrepr::{FromBytes, ToBytes};
+
+#[derive(Clone)]
+pub struct CasperPublicKey(pub casper_types::PublicKey);
+
+impl CasperPublicKey {
+    pub fn from_bytes(bytes: &[u8]) -> Result<Self, CryptoError> {
+        let (public_key, _remainder) =
+            casper_types::PublicKey::from_bytes(bytes).map_err(|_e| {
+                CryptoError::Serialization {
+                    context: "public key",
+                }
+            })?;
+        Ok(Self(public_key))
+    }
+
+    #[allow(unused)]
```

Comment:
Same as with `FromBytes` - this would make Kairos more entangled with Casper's code :roll_eyes:. I could reimplement `ToBytes` trait, but it does not seem useful at this point.

Reply:
Hmm... but if you aren't using it, do we need it for anything?